core/server/master/src/main/java/alluxio/master/meta/MetaMasterConfigurationServiceHandler.java

```diff
@@ -127,4 +129,17 @@ public void removePathConfiguration(RemovePathConfigurationPRequest request,
         return RemovePathConfigurationPResponse.getDefaultInstance();
       }, "removePathConfiguration", "request=%s", responseObserver, request);
   }
+
+  @Override
+  public void updateConfiguration(
+      UpdateConfigurationPRequest request,
+      StreamObserver<UpdateConfigurationPResponse> responseObserver) {
+    RpcUtils.call(LOG, () -> {
+      Map<String, Boolean> result
+          = mMetaMaster.updateConfiguration(request.getPropertiesMap());
```

Comment:
nit: wrong linebreak

https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break

```suggestion
      Map<String, Boolean> result =
            mMetaMaster.updateConfiguration(request.getPropertiesMap());
```

Reply:
Yeah